stock_by_warehouse/static/src/scss/main.scss

```diff
@@ -1,3 +1,8 @@
 .js_product_warehouse, .js_available_info{
     cursor: pointer;
-}
\ No newline at end of file
+}
+
+.popover-body{
+    overflow: auto;
+    height: 400px;
```

Comment:
**Add a new class or div and changed it instead .popover-body**
![image](https://user-images.githubusercontent.com/35231827/155369118-29c737a0-f463-422d-846a-4854da5c6280.png)

otherwise, you are going to modify all popovers to 400px
https://getbootstrap.com/docs/4.0/components/popovers/
![image](https://user-images.githubusercontent.com/35231827/155369782-d961556d-26e9-4d9a-b361-d2156151c613.png)

Reply:
Done, please review